src/server/swagger_docs/vscode.dev_powerplatform_v1.0.yml

```diff
@@ -0,0 +1,925 @@
+          description: "Studio SessionId - this serves as the correlationId between Studio and VSCode extension"
+          required: false
+          schema:
+            type: string
+        - name: "client"
+          in: "query"
+          description: "Name of the client from where vscode is launched. This is used only for analytics."
+          required: false
+          schema:
+            type: string
+          example: PowerPagesStudio
+      responses:
+        "200":
+          description: OK
+  /portal/contentsnippets/{contentsnippetId}:
+    get:
+      tags:
+        - portal
+      description: "Edit a specific contentsnippet"
+      parameters:
+        - name: "contentsnippetId"
+          in: "path"
+          description: "ID of the contentsnippet"
+          required: true
+          schema:
+            type: string
+            format: guid
+        - name: "orgUrl"
+          in: "query"
+          description: "Dataverse OrgUrl"
+          required: true
+          schema:
+            type: string
+          example: https://contoso.crm.dynamics.com
+        - name: "dataSource"
+          in: "query"
+          description: "Location where the portal data is saved"
+          required: true
+          schema:
+            type: string
+          examples:
+            dataverse:
+              value: dataverse
+              summary: if the data is saved in dataverse
+            github:
+              value: github
+              summary: if the data is saved in github
+        - name: "schema"
+          in: "query"
+          description: "Schema file name of the app"
+          required: true
+          schema:
+            type: string
+          examples:
+            portal old schema:
+              value: portalDataverseOldDataModelSchema
+              summary: old portal schema
+            portal new schema:
+              value: portalDataverseNewDataModelSchema
+              summary: new portal schema
+        - name: "websiteId"
+          in: "query"
+          description: "ID of the website"
+          required: true
+          schema:
+            type: string
+            format: guid
+        - name: "websiteName"
+          in: "query"
+          description: "Name of the portal you want to edit"
+          required: false
+          schema:
+            type: string
+            default: site
+        - name: "clientSessionId"
+          in: "query"
+          description: "Studio SessionId - this serves as the correlationId between Studio and VSCode extension"
+          required: false
+          schema:
+            type: string
+        - name: "client"
+          in: "query"
+          description: "Name of the client from where vscode is launched. This is used only for analytics."
+          required: false
+          schema:
+            type: string
+          example: PowerPagesStudio
+      responses:
+        "200":
+          description: OK
+  /portal/contentsnippets:
```

Comment:
not a supported sceanrios in V1.0 - this files seems incorrect - @ritikramuka - lets simplyfy this to a single swagger file with current supported APIs